src/domains/idpay-app/99_variables.tf

```diff
@@ -164,6 +164,12 @@ variable "pdv_tokenizer_url" {
   description = "PDV uri. Endpoint for encryption of pii information."
 }
 
+variable "pdv_timeout_sec" {
+  type        = number
+  description = "PDV timeout (sec)"
+  default     = 15
```

Comment:
@dariopelliccioli Maybe we should define the variable and set different values per environment

Reply:
@GiovanniMancini Added specific values for environment